packages/next/src/middleware/request.ts

```diff
@@ -60,7 +56,16 @@ export class MiddlewareRequest extends Request {
    */
   async next(options?: NextOptions): Promise<MiddlewareResponse> {
     this.applyHeaders()
-    const response = await this.context.next(options)
+    let response = await this.context.next(options)
```

Comment:
[dust] I imagine this can stay as `const`.

Reply:
It can't as it's potentially getting redefined below in the if statement.

Reply:
Ahh, didn't see that. 🙃